politeiawww/api/v1/v1.go

```diff
@@ -185,18 +187,32 @@ const (
 	PropVoteStatusDoesntExist   PropVoteStatusT = 5 // Proposal doesn't exist
 
 	// User edit actions
-	UserEditInvalid                         UserEditActionT = 0 // Invalid action type
-	UserEditExpireNewUserVerification       UserEditActionT = 1
-	UserEditExpireUpdateKeyVerification     UserEditActionT = 2
-	UserEditExpireResetPasswordVerification UserEditActionT = 3
-	UserEditClearUserPaywall                UserEditActionT = 4
-	UserEditUnlock                          UserEditActionT = 5
-	UserEditDeactivate                      UserEditActionT = 6
-	UserEditReactivate                      UserEditActionT = 7
+	UserManageInvalid                         UserManageActionT = 0 // Invalid action type
+	UserManageExpireNewUserVerification       UserManageActionT = 1
+	UserManageExpireUpdateKeyVerification     UserManageActionT = 2
+	UserManageExpireResetPasswordVerification UserManageActionT = 3
+	UserManageClearUserPaywall                UserManageActionT = 4
+	UserManageUnlock                          UserManageActionT = 5
+	UserManageDeactivate                      UserManageActionT = 6
+	UserManageReactivate                      UserManageActionT = 7
 
 	// Authorize vote actions
 	AuthVoteActionAuthorize = "authorize" // Authorize a proposal vote
 	AuthVoteActionRevoke    = "revoke"    // Revoke a proposal vote authorization
+
+	// User notification types
+	NotificationMyProposalInvalid      NotificationT = 0
```

Comment:
I think there was talk of adding in-browser notifications once websockets was hooked up.  Should we go ahead and make it explicit that these are email notifications by using a type `EmailNotificationT` instead of `NotificationT`?